interfaces/apparmor/spec_test.go

```diff
@@ -132,6 +132,27 @@ func (s *specSuite) TestAddSnippet(c *C) {
 	c.Assert(s.spec.SecurityTags(), DeepEquals, []string{"snap.demo.command", "snap.demo.service"})
 }
 
+// AddDeduplicatedSnippet adds a snippet for the given security tag.
+func (s *specSuite) TestAddDeduplicatedSnippet(c *C) {
+	restore := apparmor.SetSpecScope(s.spec, []string{"snap.demo.command", "snap.demo.service"})
+	defer restore()
+
```

Comment:
shouldn't the test also present some non dedup snippet to show the interaction of the two kinds?

Reply:
Sure, I'll add some.

Reply:
Added as a new test just below this one.